docs/app/Examples/modules/Dimmer/States/DimmerExampleDisabled.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Dimmer, Segment } from 'semantic-ui-react'
+
+const DimmerExampleDisabled = () => (
+  <Segment>
+    <Dimmer disabled />
```

Comment:
Agreed, especially since it does not override the `active` prop.  Let's remove it.